instances/measuredSignalType/betaActivity.jsonld

```diff
@@ -0,0 +1,22 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/measuredSignalType/betaActivity",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/measuredSignalType",
+  "definition": "Beta activity is an electromagnetic oscillation in the frequency range of 8–12 Hz arising from synchronous and coherent (in phase / constructive) electrical activity. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Beta_wave)]",
```

Comment:
"Beta activity is a neural oscillation in the mid frequency range (typically between 12-30 Hz) arising from synchronous and coherent electrical activity in the brain. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Beta_wave)]"

@apdavison would you agree with this definition?